docs/sdkx-react-native/support-tools.mdx

```diff
@@ -0,0 +1,154 @@
+---
+sidebar_position: 20
+title: Helpshift APIs
+description: "Helpshift provides a way to use each functional element of the SDK separately based on your requirements."
+---
+
+import {
+  Admonition,
+  CodeBlock,
+  Tabs,
+  TabItem,
+  LatestSdkVersion,
+  Centered,
+  Image,
+  Intro,
+  SideBySide,
+  DownloadButton,
+  Steps,
+  Step,
+} from "@site/src/components/forDocs";
+
+# Helpshift APIs {#support-tools}
+
+<Intro>
+
+Helpshift provides a way to use each functional element of the SDK separately based on your requirements.
+You can call only specific screens from the SDK based on the functionality you want to expose in your app.
+
+</Intro>
+
+<Admonition type="info" title="Note">
+
+All the public APIs in the SDK should be called after initializing the SDK via [Helpshift.install() API](/sdkx-react-native/getting-started#start-using)
+
+</Admonition>
+
+## Integrating Contact Us & In App Messaging {#conversation-view}
+
+<Centered width="major">
+
+![newConversation.png](/static/books/sdkx_android/show_conversation.png "newConversation.png")
+
+</Centered>
+
+You can use this API call to allow a user to directly send feedback or start a new conversation without having to first view FAQs.
+
+This API call will show the conversation screen. Once a conversation is started, it will continue until it is resolved or rejected by the agent on Helpshift Dashboard.
+
+To configure SDK, you can pass various config options as a parameter. Check out more configuration options [here].(/sdkx-react-native/sdk-configuration)
+
+Example :
+
+```
+//Create your config here.
+const config = {};  
+
+// pass the config in the api
+showConversation(config);
```

Comment:
we should have import calls for show conversation all the API's because native we only import once, but we try to import each function separately.

Reply:
Updated.